src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dayRent = 40;
+  let totalRental = 0;
+
+  if (days >= 7) {
+    totalRental = dayRent * days - 50;
```

Comment:
Avoid using literals in expressions. Define constants with descriptive names for the discount values and the thresholds for the number of days to qualify for those discounts, as per the checklist item 2.